lnbits/utils/exchange_rates.py

```diff
@@ -289,8 +295,8 @@ async def get_fiat_rate_satoshis(currency: str) -> float:
     return float(100_000_000 / (await btc_price(currency)))
 
 
-async def fiat_amount_as_satoshis(amount: float, currency: str) -> int:
-    return int(amount * (await get_fiat_rate_satoshis(currency)))
+async def fiat_amount_as_satoshis(amount: float, currency: str) -> float:
+    return float(amount * (await get_fiat_rate_satoshis(currency)))
```

Comment:
Satoshis need to be integers, lnbits doesn't support anything below 1 sat.

Reply:
I change to :


![image](https://user-images.githubusercontent.com/99406266/173360992-157e4af2-0b92-4d28-8014-99fa9812cc90.png)